netstate/routes/netstate-routes.go

```diff
@@ -0,0 +1,125 @@
+// Copyright (C) 2018 Storj Labs, Inc.
+// See LICENSE for copying information.
+
+package routes
+
+import (
+	"encoding/json"
+	"fmt"
+	"net/http"
+
+	"github.com/julienschmidt/httprouter"
+	"go.uber.org/zap"
+
+	"github.com/storj/storage/boltdb"
```

Comment:
import statement

Reply:
we might need to change something about your local dev environment to fix this? i don't know any things